test/remote-replication.test.js

```diff
@@ -0,0 +1,90 @@
+describe('Remote Replication', function() {
+  beforeEach(function(done) {
+    var test = this;
+
+    test.serverApp = loopback();
+    test.serverApp.use(function(req, res, next) {
+      console.log(req.method, req.url);
+      next();
+    });
+    test.serverApp.use(loopback.rest());
+    test.serverApp.enableAuth();
+    test.clientApp = loopback();
+    this.serverApp.set('legacyExplorer', false);
+    var settings = {
+      base: 'PersistedModel',
+      trackChanges: true,
+      dataSource: null,
+      http: {
+        path: 'my-model'
+      }
+    };
+    test.ServerModel = test.serverApp.model('ServerModel', settings);
+    test.ClientModel = test.clientApp.model('ClientModel', settings);
```

Comment:
You should use unique model names. Registering multiple models with the same name causes all sorts of weird behaviour.